ui/templates/data.html

```diff
@@ -11,13 +11,6 @@
 <div id="data">
```

Comment:
I agree but this is outside the scope of the PR. By the way this is where the datatable goes when you visit /repositories/repo/data/